interp/src/structures/names.rs

```diff
@@ -82,6 +100,28 @@ pub struct QualifiedInstanceName {
     name: Id,
 }
 
+impl QualifiedInstanceName {
+    pub fn as_id(&self) -> Id {
+        let mut acc = String::new();
+        match self.prefix.len() {
+            0 => {
+                unreachable!("This should never happen")
```

Comment:
`unreachable` already implies it shouldn't happen. Maybe say something along the lines "Fully qualified name is empty"